src/tracks/SampleTrack.cpp

```diff
@@ -352,20 +356,30 @@ void SampleTCOView::contextMenuEvent( QContextMenuEvent * _cme )
 	if( fixedTCOs() == false )
 	{
 		contextMenu.addAction( embed::getIconPixmap( "cancel" ),
-					tr( "Delete (middle mousebutton)" ),
-						this, SLOT( remove() ) );
+					tr( individualTCO
+						? "Delete (middle mousebutton)"
+						: "Delete selection (middle mousebutton)" ),
+						[this](){ contextMenuAction( Remove ); } );
```

Comment:
This and all the other lambdas are indented as if they are part of the `tr()` function. This is confusing and should be de-dented.